mantidproject/mantid
# Period Button Muons

**Description of work.**
Updated nexus reader to read period information for muons. 
Added a new widget for period information to display as a table.
Added a button to the grouping tab in Muon Analysis to display the new widget

**To test:**
Open Muon Analysis
Go to the grouping tab
Click the periods button (table should be empty)
Load current run (or some other data)
Close the table and re-open
The table should be populated

Fixes #30905 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
